certora/specs/SafeMath.spec

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+definition WAD() returns uint256 = 10^18;
+
+function summaryWMulDown(uint256 x,uint256 y) returns uint256 {
+    // Safe require because the reference implementation would revert.
```

Comment:
Implemented in https://github.com/morpho-org/pre-liquidation/pull/89/commits/2a69266b85fc2101a5712376b669aea48e5df89f.

Reply:
This results in the rule `ltvAgainstLltvEquivalentCheck` actually not checking an equivalence _stricto sensu_ because `ltv` can be arbitrarily big since it's a `mathint`.

Reply:
I don't get it, `ltv` cannot be arbitrarily large if it is essentially defined as `borrowed / collateralQuoted` and that we have `borrowed <= lltv * collateralQuoted`. Intuitively, it bounds `ltv` below `lltv` (though there are some computation details to sort).

I think that having the equivalence is really important here, it avoids mistakes that would go undetected, such as the one above in the review

Reply:
May be related to the parenthesis issue that you picked up!

Reply:
Fixing the math restores the equivalence.

Reply:
Done in https://github.com/morpho-org/pre-liquidation/pull/89/commits/c715feb778826566a40deb4543be216a1c98523e.